GMG/Core/Services/RecordManager.swift

```diff
@@ -86,6 +86,24 @@ final class RecordManager {
         return audioRecorder.url
     }
 
+    private func activateAudioSession() throws {
+        let audioSession: AVAudioSession = AVAudioSession.sharedInstance()
+
+        try audioSession.setCategory(
+            .playAndRecord,
+            mode: .default,
+            options: [.defaultToSpeaker]
+        )
+        try audioSession.setActive(true)
+    }
+
+    private func deactiveAudioSession() throws {
```

Comment:
위쪽 메서드는 activate인데 아래쪽은 deactive네요! deactivateAudioSession으로 바꾸면 좋을 것 같습니다!
```suggestion
    private func deactivateAudioSession() throws {
```